src/index.js

```diff
@@ -21,6 +21,7 @@ export {
 
 export { default as Provider } from "./Provider"
 export { default as inject } from "./inject"
+export { disposeOnUnmount } from "./disposeOnUnmount"
```

Comment:
For naming, the name is clear, but quite long. Would there be a downside to calling it `onUnmount`?

Reply:
Not entirely sure though

Reply:
I'm open for suggestions, so up to you 😁

Reply:
I'm open for suggestions, so up to you 😁

Reply:
I'm just weary of how it'd look on the non decorator version
onUnmount(reaction...)
makes it sound like the reaction will trigger on unmount or something

Reply:
Ok, let's keep it as is! If it is to long people can always alias during import :)